### PR TITLE
Replace Icons with Vector Assets

### PR DESCRIPTION
**Pull request by pluddy**
This PR updates all symbols in the app to vector assets.  This should result in better scaling for retina resolutions, and future reuse at different sizes.

Note:  SwiftUI seems to have a problem right now with resizing images inside Picker items.  I am injecting an NSImage with proper size as a workaround.

_screenshot_
image

### CODE REVIEW

File: ControlRoom/Controllers/SimulatorsController.swift [hunk 1]
```diff
@@ -56,9 +56,11 @@ class SimulatorsController: ObservableObject {
         self.preferences = preferences
         loadSimulators()
 
-        preferences.objectWillChange.sink(receiveValue: { [weak self] in
-            self?.filterSimulators()
-        }).store(in: &cancellables)
+        preferences.objectDidChange
```
Comment (pluddy) [inline ControlRoom/Controllers/SimulatorsController.swift]: this fixes a problem where the powered devices filter was flipped

**Comment by davedelong** [comment]
These look fabulous! Are these icons you created yourself? If so, can you include the source files in the "Working" directory and delete the `.pxd` documents for the old icons?

**Comment by pluddy** [comment]
> These look fabulous! Are these icons you created yourself? If so, can you include the source files in the "Working" directory and delete the `.pxd` documents for the old icons?

Thanks @davedelong! the wifi is just an SF Symbol with the opacities altered, the cell bars are actually a modified SF Symbol from the "bar chart" haha.  Ill do as you said an replace those working assets!

**Comment by twostraws** [comment]
This is awesome work! Is this good to merge in?

**Comment by pluddy** [comment]
@twostraws I'm good if you are!

**Pull request merged**